tools/text_normalization/README.md

```diff
@@ -0,0 +1,11 @@
+Text normalization system for english, e.g. 123 kg -> one hundred twenty three kilograms
+Offers prediction and evaluation on text normalization data, e.g. google text normalization dataset
+https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish.
```

Comment:
The link for the data is missing, also please add the link to the tutorial

Reply:
done